migrations/2024.08.01T15.22.49.bundles.nullable-data-items-signature.sql

```diff
@@ -0,0 +1,83 @@
+CREATE TABLE stable_data_items_new (
```

Comment:
Instead of recreating the tables, we need to see if the second answer (the one using `PRAGMA writable_schema`)  described in this StackOverflow question (https://stackoverflow.com/questions/4007014/alter-column-in-sqlite) will work.

Reply:
As we discussed, we will keep the migration this way as it's safest this way.